### PR TITLE
Added dvpe certificate issuer controller helm chart

### PR DESCRIPTION
**Pull request by cd-at-iteratec**

### CODE REVIEW

File: charts/dvpe-certificate-issuer-controller/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# dvpe-certificate-issuer-controller
+
+![Version: 0.0.1](https://img.shields.io/badge/Version-0.0.1-informational?style=flat-square)
+
+Helm chart for deploy certificate issuer controller
```
Comment (mithie) [inline charts/dvpe-certificate-issuer-controller/README.md:5]:
```suggestion
Helm chart for deploying a custom certificate issuer controller. The certificate issuer controller is a [cert-manager](https://cert-manager.io/docs/) resource managing certificate requests in a private PKI.
```

File: charts/dvpe-certificate-issuer-controller/values.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      # issuercontroller.spec.image.repository -- The docker repository to pull the service image from.
+      repository: 
+      # issuercontroller.spec.image.name -- The image name to use.
+      name: 
+      # issuercontroller.spec.image.tag -- The image version to use.
+      tag: 
+      # issuercontroller.spec.image.pullPolicy -- The default rule for downloading images.
+      pullPolicy: Always
+  externalsecrets:
+    # issuercontroller.externalsecrets.name -- The name of external secret key (dataFroms)
```
Comment (mithie) [inline charts/dvpe-certificate-issuer-controller/values.yaml:24]:
```suggestion
    # issuercontroller.externalsecrets.name -- The name of the external secret key containing the docker credentials for this deployment
```

**Review by mithie — CHANGES_REQUESTED** [review]
only found a minor issue, everything else looks fine

File: charts/dvpe-certificate-issuer-controller/README.md (hunk 1 above)
Comment (cd-at-iteratec) [reply charts/dvpe-certificate-issuer-controller/README.md]: Thanks @mithie,

I changed the documentation.
Please approve again.

**Review by mithie — APPROVED** [review]

**Pull request merged**